fares-data-build-tool/fdbt-site
# feat: term time page

# Description

-   This PR adds a new page to the school service ticket journey.

# Testing instructions

-   Pull down this branch and run the site locally. Check that the new `/termTime` page appears in the journey where it should and that it behaves as expected.

# Type of change


# Checklist:
